js/script.js

```diff
@@ -0,0 +1,187 @@
+    controllButtons(false);
+    resetGame();
+    params.question = window.prompt('How many round would you like to play?');
+    if (isNaN(params.question) || params.question.length < 1) {
+        params.roundNumber.innerHTML = 'Please, write a number';
+    } else {
+        params.winner = Math.floor(params.question * 50 / 100 + 1);
+        params.roundNumber.innerHTML = 'Number of round: ' + params.question + '  -   you will be a winner if you win ' + params.winner + ' rounds';
+    }
+    hideModal();
+});
+
+//RESET GAME
+
+function resetGame() {
+    params.humanScore = 0;
+    params.computerScore = 0;
+    params.score.innerHTML = params.humanScore + ' - ' + params.computerScore;
+    params.output.innerHTML = '';
+    params.gameResult.innerHTML = '';
+    params.eachRound = 0;
+    for (var i = 0; i < params.circleButton.length; i++) {
+        params.circleButton[i].classList.remove('circle');
+    }
+}
+
+function computerMove() {
+    var random = Math.floor(Math.random() * 3);
+    var computerChoices = ['paper', 'rock', 'scissors'];
+    return computerChoices[random];
+};
+
+//PROCESS OF GAME
+
+function playerMove(name) {
+    var computerM = computerMove();
+    params.eachRound++;
+    if ((name === 'rock' && computerM === 'paper') ||
+        (name === 'paper' && computerM === 'scissors') ||
+        (name === 'scissors' && computerM === 'rock')) {
+        params.computerScore++;
+        params.score.innerHTML = params.humanScore + ' - ' + params.computerScore;
+        var computerWon = 'computer won';
+        params.progress.push({
+            'round-nb': params.eachRound,
+            'computer-move': computerM,
+            'player-move': name,
+            'result': computerWon,
+        });
+        params.output.innerHTML = 'COMPUTER WON. You played ' + name + ', computer played ' + computerM;
+        endRound();
+    } else if ((name === 'rock' && computerM === 'scissors') ||
+        (name === 'scissors' && computerM === 'paper') ||
+        (name === 'paper' && computerM === 'rock')) {
+        params.humanScore++;
+        params.score.innerHTML = params.humanScore + ' - ' + params.computerScore;
+        var youWon = 'you won';
+        params.progress.push({
+            'round-nb': params.eachRound,
+            'computer-move': computerM,
+            'player-move': name,
+            'result': youWon,
+        });
+        params.output.innerHTML = 'YOU WON. You played ' + name + ', computer played ' + computerM;
+        endRound();
+    } else {
+        params.output.innerHTML = 'DRAW. You played ' + name + ', computer played ' + computerM;
+        var draw = 'draw';
+        params.progress.push({
+            'round-nb': params.eachRound,
+            'computer-move': computerM,
+            'player-move': name,
+            'result': draw,
+        });
+    }
+}
+
+//END ROUND
+
+function endRound() {
+    if (params.humanScore == params.winner || params.computerScore == params.winner) {
```

Comment:
jest jakiś powód, dla którego porównujesz przez `==` ? Wyżej robiłeś to przez `===`